lib/addons/p5.dom.js

```diff
@@ -188,7 +188,10 @@
       return converted;
     } else if (elt.tagName === "VIDEO" || elt.tagName === "AUDIO") {
       return new p5.MediaElement(elt);
-    } else {
+    } else if ( elt.tagName === "SELECT" ){
+      return createSelect( new p5.Element(elt) );
+    } 
```

Comment:
I don't think we should create the p5.Element here.. there's already a part in createSelect() that calls addElement() which does this. I would suggest just passing through the DOM element itself.